Libraries/LibGUI/BoxLayout.cpp

```diff
@@ -48,7 +48,7 @@ void BoxLayout::run(Widget& widget)
     should_log = true;
 #endif
     if (should_log)
-        dbgprintf("BoxLayout: running layout on %s{%p}, entry count: %d\n", widget.class_name(), &widget, m_entries.size());
+        dbgprintf("BoxLayout: running layout on %s{%p}, entry count: %lu\n", widget.class_name(), &widget, m_entries.size());
```

Comment:
these .size() are size_t, so the format specifier should be `%zu`

Reply:
Good catch, I'll change these.